pkg/packagekit/package_wix.go

```diff
@@ -173,6 +175,34 @@ func PackageWixMSI(ctx context.Context, w io.Writer, po *PackageOptions, include
 	return nil
 }
 
+// getSigntoolPath attempts to look up the location of signtool so that
+// we do not have to rely on a hard-coded signtool location that will change
+// when we upgrade to a new version of signtool.
+func getSigntoolPath() string {
+	var signtoolPath string
+	root := `C:\Program Files (x86)\Windows Kits` // restrict our lookup to a well-known location
+	fileSystem := os.DirFS(root)
+
+	if err := fs.WalkDir(fileSystem, ".", func(currentPath string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		if !d.IsDir() && strings.HasSuffix(currentPath, `x64/signtool.exe`) {
```

Comment:
This finds the first one, do you think we need to try to find the highest version?

Reply:
I think that's a good idea -- I can do that

Reply:
Installed two versions of signtool locally so I could test -- this appears to work fine. I don't love that it's string comparison, but 🤷‍♀️